bsonnumpy/bson/bson-version.h

```diff
@@ -69,7 +69,7 @@
  * BSON version, encoded as a string, useful for printing and
  * concatenation.
  */
-#define BSON_VERSION_S "1.6.3"
+#define BSON_VERSION_S "1.18.0-pre"
```

Comment:
Since this says 1.18, does that mean you copied these files from mongo-c-driver@master? We need to copy the files from mongo-c-driver@1.17.0.

Reply:
Fixed this.